dsaas-services/f8-jenkins-idler.yaml

```diff
@@ -11,7 +11,7 @@ services:
   - name: staging
     parameters:
       IMAGE: prod.registry.devshift.net/osio-prod/fabric8-services/fabric8-jenkins-idler
-- hash: a6bcdf660e96c0599e7aea9e20037afe3eb9cc2c
+- hash: 6e9fcefc0451bccbfeaa07b825d309b9dc9eb495
```

Comment:
shouldn't this go in the jenkins-proxy section?

Reply:
yes, it must be proxy did I update the wrong section??

Reply:
@jmelis it seems I did update the right section.

Reply:
You are very right! I'm sorry